aries_cloudagent/protocols/endorse_transaction/v1_0/routes.py

```diff
@@ -0,0 +1,550 @@
+    """Path parameters and validators for request taking transaction id."""
+
+    tran_id = fields.Str(
+        description="Transaction identifier", required=True, example=UUIDFour.EXAMPLE
+    )
+
+
+class AssignTransactionJobsSchema(OpenAPISchema):
+    """Assign transaction related jobs to connection record."""
+
+    transaction_my_job = fields.Str(
+        description="Transaction related jobs",
+        required=False,
+        validate=validate.OneOf(
+            [r.name for r in TransactionJob if isinstance(r.value[0], int)] + ["reset"]
+        ),
+    )
+
+
+class ConnIdMatchInfoSchema(OpenAPISchema):
+    """Path parameters and validators for request taking connection id."""
+
+    conn_id = fields.Str(
+        description="Connection identifier", required=True, example=UUIDFour.EXAMPLE
+    )
+
+
+@docs(
+    tags=["endorse-transaction"],
+    summary="Query transactions",
+)
+@querystring_schema(TransactionsListQueryStringSchema())
+@response_schema(TransactionListSchema(), 200)
+async def transactions_list(request: web.BaseRequest):
+    """
+    Request handler for searching transaction records.
+
+    Args:
+        request: aiohttp request object
+
+    Returns:
+        The transaction list response
+
+    """
+
+    context: AdminRequestContext = request["context"]
+
+    tag_filter = {}
+    post_filter = {}
+
+    try:
+        async with context.session() as session:
+            records = await TransactionRecord.query(
+                session, tag_filter, post_filter_positive=post_filter, alt=True
+            )
+        results = [record.serialize() for record in records]
+    except (StorageError, BaseModelError) as err:
+        raise web.HTTPBadRequest(reason=err.roll_up) from err
+
+    return web.json_response({"results": results})
+
+
+@docs(tags=["endorse-transaction"], summary="Fetch a single transaction record")
+@match_info_schema(TranIdMatchInfoSchema())
+@response_schema(TransactionRecordSchema(), 200)
+async def transactions_retrieve(request: web.BaseRequest):
+    """
+    Request handler for fetching a single transaction record.
+
+    Args:
+        request: aiohttp request object
+
+    Returns:
+        The transaction record response
+
+    """
+
+    context: AdminRequestContext = request["context"]
+    transaction_id = request.match_info["tran_id"]
+
+    try:
+        async with context.session() as session:
+            record = await TransactionRecord.retrieve_by_id(session, transaction_id)
+        result = record.serialize()
+    except StorageNotFoundError as err:
+        raise web.HTTPNotFound(reason=err.roll_up) from err
+
+    return web.json_response(result)
+
+
+@docs(
+    tags=["endorse-transaction"],
+    summary="For author to send a transaction request",
+)
+@querystring_schema(TranIdMatchInfoSchema())
+@querystring_schema(ConnIdMatchInfoSchema())
+@response_schema(TransactionRecordSchema(), 200)
+async def transaction_create_request(request: web.BaseRequest):
```

Comment:
For this endpoint there are inputs for a transaction id and connection id, but I don't know how to attach the transaction (was expecting a message attachment?)

E.g. I expect:
- call create schema with "auto_endorse" = "false"
- get a transaction object (signed and taa-endorsed)
- pass this object to this "create-request" endpoint